src/go/rpk/pkg/cli/cmd/redpanda/start_test.go

```diff
@@ -475,9 +464,7 @@ func TestStartCommand(t *testing.T) {
 			"--install-dir", "/var/lib/redpanda",
 		},
 		postCheck: func(fs afero.Fs, _ *redpanda.RedpandaArgs, st *testing.T) {
-			path := config.Default().ConfigFile
-			mgr := config.NewManager(fs)
-			conf, err := mgr.Read(path)
+			conf, err := new(config.Params).Load(fs)
 			require.NoError(st, err)
 			require.Exactly(st, "aws:i3xlarge:default", conf.Rpk.WellKnownIo)
```

Comment:
As a followup PR (later) it'd probably be useful to get rid of these before and postCheck functions and just be entirely driven by structs with expected state.